tests/test_package_manager/test_v1.py

```diff
@@ -280,3 +280,39 @@ def test_missing_hash(self, caplog) -> None:
 
             assert pm.verify() == 1
             assert f"Cannot find hash for {EXAMPLE_PACKAGE_ID}" in caplog.text
+
+
+def test_package_manager_add_item_dependency_support():
+    """Check PackageManager.add_packages works with dependencies."""
+    with tempfile.TemporaryDirectory() as tmpdir:
+        package_manager = PackageManagerV1(tmpdir)
+        package_manager.add_package(
+            PackageId(
+                package_type=PackageType.SKILL,
+                public_id=PublicId.from_str(
+                    "valory/abstract_round_abci:0.1.0:bafybeifh4qtjurq5637ykxexzexca5l4n6t4ujw26tpnern2swajanvhny"
```

Comment:
We cannot have this here because the `open-aea` should not be aware about the `abstract_round_abci` skill. Also direct interaction with the IPFS registry makes this an integration test and we don't want that right now because the IPFS registry cluster is a bit unstable right now. So I would suggest you add some mocking and don't use the IPFS registry

Reply:
this test will be set as integration. package name does not matter, it has to present on the IPFS for testing only (and actually presetns)

new tests will be done with mocks